invenio_webhooks/views.py

```diff
@@ -101,18 +126,58 @@ def post(self, receiver_id=None):
         # db.session.begin(subtransactions=True)
         event.process()
         db.session.commit()
-        return jsonify(**event.response), event.response_code
+        return make_response(event)
 
     def options(self, receiver_id=None):
         """Handle OPTIONS request."""
         abort(405)
 
 
+class ReceiverEventResource(MethodView):
+    """Event resource."""
+
+    @staticmethod
+    def _get_event(receiver_id, event_id):
+        """Find event and check access rights."""
+        event = Event.query.filter_by(
+            receiver_id=receiver_id, id=event_id
+        ).first_or_404()
+
+        if event.user_id != request.oauth.access_token.user_id:
+            abort(401)
+
+        return event
+
+    @require_api_auth()
+    @require_oauth_scopes('webhooks:event')
+    @error_handler
+    def get(self, receiver_id=None, event_id=None):
+        """Handle GET request."""
+        event = self._get_event(receiver_id, event_id)
+        return make_response(event)
+
+    @require_api_auth()
+    @require_oauth_scopes('webhooks:event')
```

Comment:
Would it makes sense to have a different scope for deletion in order to support the use-case where an application is allowed to submit events but not to stop them.